docs/tutorials/aws/semantic_search_with_CFN_template_for_Sagemaker.md

```diff
@@ -0,0 +1,240 @@
+map IAM role to `ml_full_access` to grant it permission. 
+You can refer to [semantic_search_with_sagemaker_embedding_model#map-backend-role](https://github.com/opensearch-project/ml-commons/blob/2.x/docs/tutorials/aws/semantic_search_with_sagemaker_embedding_model.md#22-map-backend-role) part. 
+
+The IAM role can be located in the "Lambda Invoke OpenSearch ML Commons Role Name" field on the CloudFormation template. Please refer to the screenshot in step 2.1.
+
+The default IAM role is `LambdaInvokeOpenSearchMLCommonsRole`, so you need to map this backend role `arn:aws:iam::your_aws_account_id:role/LambdaInvokeOpenSearchMLCommonsRole` to `ml_full_access`.
+
+For quick start, you can also map all roles to `ml_full_access` with wildcard `arn:aws:iam::your_aws_account_id:role/*`
+
+As `all_access` has more permission than `ml_full_access`, it's ok to map backend role to `all_acess`.
+
+
+## 2. Run CloudFormation template
+
+You can find CloudFormation template integration on AWS OpenSearch console.
+
+![Alt text](images/semantic_search/semantic_search_remote_model_Integration_1.png)
+
+For all options below, you can find OpenSearch AI connector and model id in "Outputs" of CloudFormation stack when it completes.
+
+If  you see any failure, you can find log on Sagemaker Console by searching "Log Groups" with CloudFormation stack name.
+
+### 2.1 Option1: Deploy pretrained model to Sagemaker
+
+You can deploy pretrained Huggingface sentence-transformer embedding model from [DJL](https://djl.ai/) model repo.
+
+Keep other fields as default value if not mentioned below:
+
+1. You must fill your "Amazon OpenSearch Endpoint"
+2. You can use default setting of "Sagemaker Configuration" part for quick start. If necessary, you can fine tune the values. You can find all supported Sagemaker instance type [here](https://aws.amazon.com/sagemaker/pricing/).
+3. You must leave "SageMaker Endpoint Url" as empty. By inputting this value, you will not deploy model to Sagemaker to create a new inference endpoint.
+4. You can leave "Custom Image" part as empty, it will use `djl-inference:0.22.1-cpu-full` as default value. You can find more available image from [this document](https://docs.aws.amazon.com/deep-learning-containers/latest/devguide/deep-learning-containers-images.html).
+5. You must leave "Custom Model Data Url" as empty for this option.
+6. The default value "Custom Model Environment" is `djl://ai.djl.huggingface.pytorch/sentence-transformers/all-MiniLM-L6-v2`, find all supported model in Appendix part of this doc.
+
+![Alt text](images/semantic_search/semantic_search_remote_model_Integration_2.png)
+
+
+### 2.2 Option2: Create model with your existing Sagemaker inference endpoint
+
+If you already have a Sagemaker inference endpoint, you can create remote model directly with it.
+
+Keep other fields as default value if not mentioned below:
+1. You must fill your "Amazon OpenSearch Endpoint"
+2. You must fill your "SageMaker Endpoint Url".
```

Comment:
Add "For example, https://runtime.sagemaker.us-east-1.amazonaws.com/endpoints/<Your SageMaker Model Name>/invocations" as an example of the url?